draft-kuehlewind-iab-rfc4053bis.md

```diff
@@ -417,13 +418,13 @@ other organization.
    A request for Action is a fairly serious thing.  Examples of the
    kinds of actions that may be expected are:
 
-- In the case of overlapping or related work in another
-      organization, another organization may request that the IETF align
-      its work with that of the other organization.
-- A request could be made for IETF to undertake a new work item.
-- A request could be made for IETF to stop a work item (presumably
-      because it overlaps or conflicts with other work in the
-      originating organization).
+   - In the case of overlapping or related work in another
+     organization, another organization may request that the IETF align
+     its work with that of the other organization.
+   - A request could be made for IETF to undertake a new work item.
+   - A request could be made for IETF to stop a work item (presumably
+     because it overlaps or conflicts with other work in the
+     originating organization).
```

Comment:
We should remove the indention everywhere. That's just leftover from the copy-paste of the original RFC. In particularly in list indention has a meaning in markdown.
```suggestion
- In the case of overlapping or related work in another
     organization, another organization may request that the IETF align
     its work with that of the other organization.
- A request could be made for IETF to undertake a new work item.
- A request could be made for IETF to stop a work item (presumably
     because it overlaps or conflicts with other work in the
     originating organization).
```